gradle.properties

```diff
@@ -10,7 +10,8 @@
 
 # 3rd party Versions.
 checkstyleToolVersion=7.1
-flinkVersion=1.6.0
+flinkVersion=1.7.2
+flinkScalaVersion=2.12
```

Comment:
Is there any particular reason for moving to `2.12` or is it being done opportunistically?

Reply:
It only to take advantage of latest scala version support from Flink.